src/azure-cli/azure/cli/command_modules/storage/_validators.py

```diff
@@ -337,11 +337,7 @@ def _class_name(class_type):
         return class_type.__module__ + "." + class_type.__class__.__name__
 
     def validator(cmd, namespace):
-        t_base_blob_service, t_file_service, t_blob_content_settings, t_file_content_settings = cmd.get_models(
-            'blob.baseblobservice#BaseBlobService',
-            'file#FileService',
-            'blob.models#ContentSettings',
-            'file.models#ContentSettings')
+        t_file_content_settings = cmd.get_models('file.models#ContentSettings')
 
```

Comment:
It looks that t_file_content_setting is not used in this file.

Reply:
It is used in line 345. You may need to expand the lines.